osu.Game/Overlays/BeatmapListing/BeatmapSearchFilter.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.UserInterface;
+using osu.Framework.Input.Events;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Graphics.UserInterface;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Overlays.BeatmapListing
+{
+    public class BeatmapSearchFilter<T> : TabControl<T>
+    {
+        public BeatmapSearchFilter()
+        {
+            AutoSizeAxes = Axes.Both;
+
+            if (typeof(T).IsEnum)
+            {
+                foreach (var val in (T[])Enum.GetValues(typeof(T)))
+                    AddItem(val);
+            }
+        }
+
+        protected override Dropdown<T> CreateDropdown() => null;
+
+        protected override TabItem<T> CreateTabItem(T value) => new FilterTabItem(value);
+
+        protected override TabFillFlowContainer CreateTabFlow() => new TabFillFlowContainer
+        {
+            AutoSizeAxes = Axes.Both,
+            Spacing = new Vector2(10),
+        };
+
+        protected class FilterTabItem : TabItem<T>
+        {
+            [Resolved]
+            private OverlayColourProvider colourProvider { get; set; }
+
+            private readonly OsuSpriteText text;
+
+            public FilterTabItem(T value)
+                : base(value)
+            {
+                AutoSizeAxes = Axes.Both;
+                AddRangeInternal(new Drawable[]
+                {
+                    text = new OsuSpriteText
+                    {
+                        Font = OsuFont.GetFont(size: 13, weight: FontWeight.Regular),
```

Comment:
Font size should be configurable if advanced search criteria are to be moved to the game too, as they have size 10 instead of 13:

![image](https://user-images.githubusercontent.com/20418176/74604759-39fccb00-50c1-11ea-8594-ebd232e02de2.png)

Reply:
Fixed